ecommerce/api.py

```diff
@@ -359,6 +362,9 @@ def process_cybersource_payment_response(request, order):
 
         return_message = order.state
     else:
+        log.error(
+            f"Unknown code {processor_response.state} found: transaction ID {transaction_id}, reason code {reason_code}, response messager {processor_response.message}"
```

Comment:
Maybe we want to change it a bit from code to state? but I'll let you decide. Other than this there is a typo in `message`.
```suggestion
            f"Unknown state {processor_response.state} found: transaction ID {transaction_id}, reason code {reason_code}, response message {processor_response.message}"
```